guides/v2.3/graphql/release-notes.md

```diff
@@ -17,60 +17,60 @@ These release notes can include:
-  - Set the [payment method]({{page.baseurl}}/graphql/reference/quote-payment-method.html). Supported methods include Bank Transfer, Cash on Delivery, Check/Money Order, Purchase Order, and Zero Subtotal Checkout.
-  - [Apply]({{page.baseurl}}/graphql/reference/quote-apply-coupon.html) or [remove]({{page.baseurl}}/graphql/reference/quote-remove-coupon.html) cart coupons.
-  - [Assign an email]({{page.baseurl}}/graphql/reference/quote-set-guest-email.html) to a guest cart.
-  - [Place an order]({{page.baseurl}}/graphql/reference/quote-place-order.html).
+   -  Add [simple products]({{page.baseurl}}/graphql/mutations/add-simple-products.html) to a cart.
+   -  Add [virtual products]({{page.baseurl}}/graphql/mutations/add-virtual-products.html) to a cart.
+   -  Set the [shipping address]({{page.baseurl}}/graphql/mutations/set-shipping-address.html). Address books are supported.
+   -  Set the [billing address]({{page.baseurl}}/graphql/mutations/set-billing-address.html). Address books are supported.
+   -  Set the [shipping method]({{page.baseurl}}/graphql/mutations/set-shipping-method.html). Supported methods include DHL, FedEx, Flat Rate, Free Shipping, Table Rate, UPS, and USPS.
+   -  Set the [payment method]({{page.baseurl}}/graphql/mutations/set-payment-method.html). Supported methods include Bank Transfer, Cash on Delivery, Check/Money Order, Purchase Order, and Zero Subtotal Checkout.
+   -  [Apply]({{page.baseurl}}/graphql/mutations/apply-coupon.html) or [remove]({{page.baseurl}}/graphql/mutations/remove-coupon.html) cart coupons.
+   -  [Assign an email]({{page.baseurl}}/graphql/mutations/set-guest-email.html) to a guest cart.
+   -  [Place an order]({{page.baseurl}}/graphql/mutations/place-order.html).
 
-- {:.new} **Added support for payment methods that implement [Magento Vault]({{page.baseurl}}/graphql/reference/vault.html)**
--
-- {:.new} **Added new queries and extended the functionality of others.**
+-  {:.new} **Added support for payment methods that implement [Magento Vault]({{page.baseurl}}/graphql/reference/vault.html)**
 
-  - The [`isEmailAvailable` query]({{page.baseurl}}/graphql/reference/customer.html) checks whether the specified email address has already been used to create an account.
-  - The [`cart` query]({{page.baseurl}}/graphql/reference/quote.html) can now return information set by mutations that perform cart operations, including product information, shipping and billing addresses, shipping and payment methods, and applied coupons. The query also returns calculated totals.
-  - The `customerPaymentTokens` query returns the signed-in customer's payment tokens.
+-  {:.new} **Added new queries and extended the functionality of others.**
 
-- {:.new} **Queries can now be performed as HTTP GET or POST operations.**
+   -  The [`isEmailAvailable` query]({{page.baseurl}}/graphql/reference/customer.html) checks whether the specified email address has already been used to create an account.
```

Comment:
Should this link point to https://devdocs.magedevteam.com/804/guides/v2.3/graphql/queries/customer.html?

Reply:
Fixed to point to `queries/is-email-available.html`